### ISSUE TITLE
Focus indicator should be clearly visible while navigating to Primary Button, Default Button and close button in Dialog control

### ISSUE THREAD

**Issue by lovakumar** [OfficeDev/office-ui-fabric-react#3714]
Issue #1: Focus indicator should be clearly visible while navigating to Primary Button, Default Button and close button in Dialog control. 

Issue #2: **In high contrast mode** focus indicator should be clearly visible while navigating to Primary Button, Default Button and close button in Dialog control. 

Details:
Button with icon props
Primary Button
Default Button
Close (X) button on the right top corner of Dialog control

**Comment by betrue-final-final** [comment]
I'm not seeing issue 1. Do you have repro steps?
![image](https://user-images.githubusercontent.com/20363037/34883616-ef4fb746-f76e-11e7-9692-416d30298e79.png)


Issue 2 is a duplicate of issue #3419

**Comment by lovakumar** [comment]
We have verified issue #1 again and seems like it works fine in Edge but not in IE. Can you please verify it in IE browser.

**Comment by betrue-final-final** [comment]
I'm not seeing that in IE. If it's older than IE11, then it's officially not supported, but it should still work there too.
![image](https://user-images.githubusercontent.com/20363037/35341102-bb3574a4-00d9-11e8-8ef8-b7a64aac54d6.png)
